crates/bevy_pbr/src/meshlet/cull_meshlets.wgsl

```diff
@@ -65,7 +68,11 @@ fn cull_meshlets(@builtin(global_invocation_id) cluster_id: vec3<u32>) {
     // TODO: Faster method from https://vkguide.dev/docs/gpudriven/compute_culling/#frustum-culling-function
     for (var i = 0u; i < 6u; i++) {
         if !meshlet_visible { break; }
-        meshlet_visible &= dot(view.frustum[i], culling_bounding_sphere_center) > -culling_bounding_sphere_radius;
+        meshlet_visible &= sphere_is_inside_frustum_plane(
```

Comment:
Was this name changed from sphere_intersects? If so, I'd prefer sphere_intersects for consistency with the main code. 'Sphere is inside' suggests the entire sphere is contained within, but that isn't what the frustum intersection means, right?

Reply:
I think I wrote "inside" because "intersects a plane" is wrong: it's not whether the sphere intersects the plane, but whether it's in the positive half-space defined by the plane. I'll make it `sphere_intersects_plane_half_space`.